README.md

```diff
@@ -1,2 +1,44 @@
 # OC-Test
 OpenShift Testing
+
+Pipeline setup steps
+
+1) Set up Jenkins project  
+##create new project named test-app-pipeline  
```

Comment:
Looks like the markdown syntax isn't correct. Please use a markdown previewer.

Reply:
Fixed!